### DISCUSSION TITLE
Infinispan 15.0.0 final download link  -  widllfy Module

### DISCUSSION THREAD

**Discussion by Satishshelake** [infinispan/infinispan#12264] [Show and tell]
Wildfly Module - Infinispan 15.0.0 download link in not working on
https://infinispan.org/download/

**Comment by tristantarrant** [comment]
WildFly modules were removed from Infinispan 15. https://issues.redhat.com/browse/ISPN-14786
We should fix the website to hide that link

**Reply by Satishshelake** [reply]
@tristantarrant  When will be available for download?  Couldn't infinispan support widlfly module from 15 version?

**Reply by tristantarrant** [reply]
We have decided to stop building WildFly modules. You should use the Infinispan components included by WildFly itself.
